lib/util.js

```diff
@@ -452,7 +452,7 @@ module.exports.serverSwaggerPluginId = 'org.zowe.zlux'
 module.exports.agentSwaggerPluginId = 'org.zowe.zlux.agent';
 
 module.exports.getCookieName = (port) => {
-  const zoweInstanceId = process.env['ZOWE_INSTANCE'];
+  const zoweInstanceId = process.env['ZWE_zowe_cookieIdentifier'];
```

Comment:
This needs to be changed.
get rid of the isHaMode conditional.
Instead just, cookieName='connect.sid.'+cookieIdentifier
which means you need to change the funciton arg to pass in cookieIdentifier.

Reply:
resolved